src/app/static/src/tests/utils.test.ts

```diff
@@ -389,59 +388,12 @@ describe("utils", () => {
         expect(getFilenameFromImportUrl("http://a/b.csv?c=d")).toBe("b.csv");
     });
 
-    it("updateForm merges old and new form metadata as expected", () => {
-        const oldForm: DynamicFormMeta = {
-            controlSections:[
-                {
-                    label: "old-section-1",
-                    controlGroups: [
-                        {
-                            label: "old-group-1-1",
-                            controls: [
-                                {
-                                    name: "control-1",
-                                    type: "number",
-                                    required: true,
-                                    value: 1
-                                },
-                                {
-                                    name: "control-2",
-                                    type: "select",
-                                    required: true,
-                                    value: 2
-                                }
-                            ]
-                        }
-                    ]
-                },
-                {
-                    label: "section-2",
-                    controlGroups: [
-                        {
-                            label: "group-2-1",
-                            controls: [
-                                {
-                                    name: "control-3",
-                                    type: "number",
-                                    required: false,
-                                    value: 3
-                                }
-                            ]
-                        },
-                        {
-                            label: "group-2-2",
-                            controls: [
-                                {
-                                    name: "control-4",
-                                    type: "select",
-                                    required: false,
-                                    value: 4
-                                }
-                            ]
-                        }
-                    ]
-                }
-            ]
+    it("writeOptionsIntoForm writes options into form metadata as expected", () => {
```

Comment:
Might want to do a unit test for `checkOptionsValid` by itself too?

Reply:
Thanks yes! Have added that now